### DISCUSSION TITLE
LDPC5GDecoder Exception

### DISCUSSION THREAD

**Discussion by nadh-amar** [NVlabs/sionna#260] [General]
Hi,
While running the SIONNA code for  End-to-end Learning with Autoencoders, I am facing the error :

InvalidArgumentError: Exception encountered when calling layer 'e2e_system_conventional_training_5' (type E2ESystemConventionalTraining).

Detected unsupported operations when trying to compile graph ldpc5g_decoder_3_while_body_1358924_const_0[] on XLA_GPU_JIT: RaggedRange (No registered 'RaggedRange' OpKernel for XLA_GPU_JIT devices compatible with node {{node ldpc5g_decoder_3/while/RaggedWhere/RaggedRange}}){{node ldpc5g_decoder_3/while/RaggedWhere/RaggedRange}}
The op is created at: 
File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
.
.
.
.
File "/usr/local/lib/python3.10/dist-packages/sionna/fec/ldpc/decoding.py", line 980, in dec_iter
File "/usr/local/lib/python3.10/dist-packages/sionna/fec/ldpc/decoding.py", line 654, in _cn_update_phi
	tf2xla conversion failed while converting ldpc5g_decoder_3_while_body_1358924_const_0[]. Run with TF_DUMP_GRAPH_PREFIX=/path/to/dump/dir and --vmodule=xla_compiler=2 to obtain a dump of the compiled functions.

I have not made any changes to the code. May how I can resolve this issue?

Thank you!

**Comment by SebastianCa** [comment]
Hi,
I cannot reproduce your error with Python 3.10 and Tensorflow 2.13.
To provide further support, could you please let us know which version of Sionna and Tensorflow you are using?

**Reply by nadh-amar** [reply]
Hi..
I am using Tensorflow : 2.14.0
Sionna : 0.15.1
python: 3.10.12

**Comment by SebastianCa** [comment]
I can reproduce the error using TensorFlow 2.14. 
We will fix the issue with the next release of Sionna. As a workaround, can you please downgrade to TF 2.13.

**Reply by nadh-amar** [reply]
Thank you so much. It seems to have fixed the problem :)
